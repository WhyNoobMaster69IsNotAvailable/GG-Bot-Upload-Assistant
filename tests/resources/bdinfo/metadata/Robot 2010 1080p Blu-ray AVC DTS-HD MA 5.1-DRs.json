{
    "upload_media": "/data/downloads/FullDisk/Robot 2010 1080p Blu-ray AVC DTS-HD MA 5.1-DRs/",
    "raw_video_file": "/data/downloads/FullDisk/Robot 2010 1080p Blu-ray AVC DTS-HD MA 5.1-DRs/BDMV/STREAM/00003.m2ts",
    "raw_file_name": "Robot 2010 1080p Blu-ray AVC DTS-HD MA 5.1-DRs",
    "largest_playlist": "00003.MPLS",
    "mediainfo": "/app/temp_upload/c7c95f338790ebdc1be6df838af0a56e74d4000ab7c428cdd132bffb400c42f0/mediainfo.txt",
    "bdinfo_output_split": "b'Please wait while we scan the disc...\n# Group Playlist File Length Estimated Bytes Measured Bytes \n\n1 1 00003.MPLS 02:56:59 45,854,263,296 - \n2 1 01003.MPLS 02:56:59 45,854,263,296 - \n3 2 00002.MPLS 00:00:45 180,129,792 - \n4 2 01002.MPLS 00:00:45 180,129,792 - \n'"
}
