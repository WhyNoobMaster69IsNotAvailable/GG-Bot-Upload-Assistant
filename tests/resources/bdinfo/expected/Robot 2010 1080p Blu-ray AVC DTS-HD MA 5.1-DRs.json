{
    "video": [
        {
            "codec": "AVC",
            "bitrate": "27985 kbps",
            "resolution": "1080p",
            "fps": "23.976 fps",
            "aspect_ratio": "16:9",
            "profile": "High Profile 4.1"
        }
    ],
    "audio": [
        {
            "language": "Hindi",
            "codec": "DTS-HD Master Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "4099 kbps",
            "bit_depth": "24-bit"
        },
        {
            "language": "Hindi",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -31dB"
        }
    ],
    "title": "ROBOT",
    "label": "Robot 2010 1080p Blu-ray AVC DTS-HD MA 5.1-DRs",
    "size": 42.96958874259144,
    "playlist": "00003.MPLS",
    "length": "2:56:59"
}